webpack/assets/javascripts/react_app/components/Layout/components/TaxonomySwitcher/__snapshots__/TaxonomyDropdown.test.js.snap

```diff
@@ -144,7 +304,23 @@ exports[`TaxonomyDropdown rendering rendering 1`] = `
     onClick={[Function]}
     sendRef={[Function]}
   >
-    org2
+    <Grid
+      hasGutter={true}
+    >
+      <GridItem
+        span={10}
+        style={
+          Object {
+            "textAlign": "left",
+          }
+        }
+      >
+        org2
+      </GridItem>
+      <GridItem
+        span={1}
+      />
+    </Grid>
```

Comment:
@MariaAga thoughts about removing this snap file and the test that generated it?